admin/bootstrap.py

```diff
@@ -53,6 +56,8 @@ def run_command(command):
     popen = subprocess.Popen(command,
                              stdout=subprocess.PIPE,
                              stderr=subprocess.STDOUT)
+    if popen.stdout is None:
+        raise RuntimeError("Should never happen: None stdout")
```

Comment:
If you raised `subprocess.CalledProcessError` here, the [one place](https://github.com/freedomofpress/securedrop/blob/1d50b6ecd4bc0cfe6c803630b7fb4f8d39bc3810/admin/bootstrap.py#L146) `run_command` is used would handle it with a more informative error message. If you don't like the idea of commandeering `CalledProcessError`, we could catch `RuntimeError` there too. Either way, maybe we could include the command in the exception message, to help troubleshooting in case we do use `run_command` in another context.

Reply:
This is not really a "normal" exception that you are supposed to catch and act on. A `None` stdout means that something is really wrong and the server/script can't run properly ie. a crash is the goal here (hence it says "should never happen"). I had to add this to fix type errors (but it also makes sense). 
Another option would be to use a special exception that's like `ImpossibleStateError` or something like that for these situations.

Reply:
My goal here is to not have an admin told `Should never happen: None stdout` while installing or updating SecureDrop, because that will mean absolutely nothing to them, while eroding confidence in the project. "If it should never happen, why am I seeing it?" If it can really never happen, and you're just adding it to satisfy mypy, I think `type: ignore` would be better than introducing the possibility of this. If you prefer the message, I'd like to at least give them and us something that will help troubleshoot the problem. "Error running  _command_" would be OK, or catching it where `run_command` is invoked and presenting something more specific, as done in [`install_apt_dependencies`](https://github.com/freedomofpress/securedrop/blob/1d50b6ecd4bc0cfe6c803630b7fb4f8d39bc3810/admin/bootstrap.py#L152).

Reply:
I see - I changed it to an EnvironmentError with a better error message.